modules/backend/widgets/form/partials/_field_checkbox.php

```diff
@@ -1,18 +1,14 @@
 <!-- Checkbox -->
 <div class="checkbox custom-checkbox" tabindex="0">
-    <input
-        type="hidden"
-        name="<?= $field->getName() ?>"
-        value="0"
-        <?= $this->previewMode ? 'disabled="disabled"' : '' ?>>
```

Comment:
Pretty sure this breaks this field, if a checkbox isn't selected then the field name will be absent from the request entirely which is why we need this hidden field with a value of 0 here before the checkbox input itself.

Reply:
@LukeTowers Wouldn't the field be missing from the post since its type is "hidden" ?

Reply:
No, the entire reason for hidden fields is so that they are hidden from the user but included in the POST

Reply:
Fair enough, it's back in.